aladdin/bash/container/get-dashboard-url/get-dashboard-url

```diff
@@ -1,31 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
the dashboard isn't available by default in k3d. i've moved that functionality to an internal plugin